rubric.md

```diff
@@ -0,0 +1,69 @@
+# So you want the IWG to run a service…
+
+⚠️  This document is maintained by the IWG and is likely to change as we learn
+from the process of actually carrying out our mandate.
+
+## What does "run a service" mean?
+
+The IWG has a few roles in running services:
+
+1. It holds administrative keys and delegates access to the current active
+   admins, if needed.
+2. It keeps services running, monitors availability, performs security updates,
+   and does other "keep the lights on" tasks.
+3. It identifies breakages caused by upstream changes and attempts to have them
+   fixed.  The IWG does not perform development tasks (like rewriting custom
+   software).  See "[When Things Break](#When Things Break)" below.
+4. It writes internal documentation (playbooks) for how to manage services.
+   This covers generic documentation of the hosting platform and
+   service-specific troubleshooting steps.
+
+The IWG expressly *does not* do the following:
+
+1. moderate user content
+2. develop new features
+3. fix non-trivial bugs
+4. write documentation for users
+
+## What kind of services does IWG run?
+
+There are two categories of system run by IWG:
+
+1. **TPF services**: those being used for the direct benefit of TPF
```

Comment:
Suggestion: Add examples to this section.

Reply:
Done.